Add SharePoint Online app+user-only note to entitlement management resource APIs (#10006)
microsoftgraph/microsoft-graph-docs-contrib: api-reference/v1.0/api/accesspackagecatalog-list-resources.md

[!INCLUDE [rbac-entitlement-catalog-reader](../includes/rbac-for-apis/rbac-entitlement-management-catalog-reader-apis-read.md)]

> [!NOTE]

> > SharePoint Online resources are returned only when this API is called with delegated (app+user) permissions. This scenario isn't supported with application-only permissions. To retrieve SharePoint site information in an app-only context, use the [sites: getAllSites](site-getallsites.md) API, which returns equivalent site information.

Reviewer: Hello @jconley76, for consistency with the Microsoft Graph standards, please update the note.

```suggestion
> SharePoint Online resource environments are returned only when this API is called with delegated permissions. This scenario isn't supported with application permissions. To retrieve SharePoint site information using application permissions, use the [sites: getAllSites](../api/site-getallsites.md) API, which returns equivalent site information.
```

Author: Applied, thanks. One tweak: I kept "resources" rather than "resource environments" here, since this page lists accessPackageResource objects. Wording and link otherwise match your suggestion.
